client/web/src/routes/home/home.styles.ts

```diff
@@ -1,6 +1,40 @@
 import { css } from '@microsoft/fast-element';
 
 export const HomeStyles = css`
-  :host {
-  }
+:host {
+  background: var(--fill-color);
+  color: var(--neutral-foreground-rest);
+}
+
+.wrapper {
+  background: var(--fill-color);
+  display: flex;
+}
+
+.column {
+  flex: 1;
+  padding: calc(var(--base-horizontal-spacing-multiplier) * 3px);
+}
+
```

Comment:
We shouldn't add these styles needed for the preview - it's a blank app seed after all